NumRush.py

```diff
@@ -387,8 +388,13 @@ def read_highscore(self):
         highscore = [0]
         file_path = os.path.join(get_activity_root(), 'data', 'highscore')
         if os.path.exists(file_path):
-            with open(file_path, "r") as fp:
-                highscore = fp.readlines()
+            try:
+                with open(file_path, "r") as fp:
+                    highscore = fp.readlines()
+                return int(highscore[0])
+            except ValueError or IndexError as e:
```

Comment:
Any particular reason for changing what you used before?

Reply:
Also, the exception handing syntax doesn't seem to be correct. 
```py
>>> try:
...     raise ValueError()
... except IndexError or ValueError:
...     print("caught!")
...
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
ValueError
```

Reply:
> Yea, there's a possibility to get these exceptions here when a string is passed or when the list is empty. Earlier, I had tried passing the raw output of from the file without converting it to `int` and then did the type conversion in the calling function(`load_highscore`). So, this exception handling was required in the `load_highscore` function. Later, I moved the type conversion to `read_highscore` function but missed moving the exception handing code to `read_highscore`. So fixed it now.

I meant any reason from changing the `except (ValueError, IndexError) as e` to `except IndexError or ValueError as e`

Reply:
Oh that, not really. I guess it wouldn't make any difference. Please let me know if it's not a good practice.

Reply:
got it, my bad. Fixing it now!

Reply:
> Oh that, not really. I guess it wouldn't make any difference. Please let me know if it's not a good practice.

In Python, multiple exceptions can be caught with multiple except blocks or a single except block if you want same behaviour for various exceptions - as is the case here - and a single except block works by putting the exceptions in a `()` else it won't be caught like Srevin showed above.

Reply:
understood, I've fixed it in [f0ad072](https://github.com/sugarlabs/numberrush-activity/pull/2/commits/f0ad072a4bf8b396d02514503939dc1d8854743d)